apps/server/src/modules/board/domain/video-conference-element.do.ts

```diff
@@ -0,0 +1,27 @@
+import { BoardNode } from './board-node.do';
+import type { VideoConferenceElementProps } from './types';
+
+export class VideoConferenceElement extends BoardNode<VideoConferenceElementProps> {
+	get url(): string {
+		return this.props.url ?? '';
+	}
+
+	set url(value: string) {
```

Comment:
Maybe I'm not understanding something but it seems that url is useless here. Everything depends on the boardId/RoomId.

Reply:
I think this is also what you mentioned today.